src/SourceBuild/content/eng/init-source-only.proj

```diff
@@ -24,12 +24,13 @@
           Outputs="$(BaseIntermediateOutputPath)UnpackTarballs.complete" >
     <PropertyGroup>
       <ExternalTarballsDir>$([MSBuild]::NormalizeDirectory('$(PrereqsPackagesDir)', 'archive'))</ExternalTarballsDir>
+      <SourceBuiltArtifactsTarballPath>$(ExternalTarballsDir)$(SourceBuiltArtifactsTarballName).*$(ArchiveExtension)</SourceBuiltArtifactsTarballPath>
```

Comment:
I am not sure this glob works. I couldn't see this variable in `./artifacts/log/prep-bootstrap.binlog` of the VMR. Is there some other place where I can find it?